microcosm_pubsub/context.py

```diff
@@ -15,6 +17,10 @@ def sqs_message_context(message_dct, **kwargs):
     return context
 
 
+@defaults(
+    enable_ttl=typed(bool, default_value=True),
```

Comment:
You probably want to use our custom [boolean](https://github.com/globality-corp/microcosm/blob/develop/microcosm/config/types.py#L8) to properly handle strings